ghp_import.py

```diff
@@ -74,7 +74,8 @@ def try_rebase(remote, branch):
     (rev, ignore) = p.communicate()
     if p.wait() != 0:
         return True
-    cmd = ['git', 'update-ref', 'refs/heads/%s' % branch, rev.strip()]
+    rev = rev.strip().decode('ascii')
```

Comment:
I think I'm right in saying that `rev` should always be a commit hash, so `ascii` or `utf-8` decoding (via `dec`) should give the same result.  I guess the only advantage of `ascii` is that it will give an error when the result is other than a hash.